doc/riverctl.1.scd

```diff
@@ -330,6 +330,10 @@ A complete list may be found in _/usr/include/linux/input-event-codes.h_
 *list-input-configs*
 	List all input configurations.
 
+*keyboard-layout* _layout_ _variant_ _model_ _options_ _rules_
+	Set the XKB layout for all keyboards. Use \_ for values you do not
+	want to set.
+
```

Comment:
This feature looks good, but I'm not a huge fan of this CLI interface. It seems to me that most users will only want to specify only the layout. A few more will want to specify options, but I suspect very few users will set the model or rules for example.

Consider also the readability of 
```
riverctl keyboard-layout de _ foo caps:swapesc _
```
vs
```
riverctl keyboard-layout -options caps:swapesc -model foo de
```

Reply:
I was a bit hesitant to do something like this initially, considering command parsing is done server side and river feels like the wrong place for making the riverctl UX nice.

Anyway, I should have this done sometime this week.